src/backend/columnar/cstore_reader.c

```diff
@@ -529,10 +529,10 @@ SelectedChunkMask(StripeSkipList *stripeSkipList, List *projectedColumnList,
 #else
 			predicateRefuted = predicate_refuted_by(constraintList, restrictInfoList);
 #endif
-			if (predicateRefuted)
```

Comment:
@jeff-davis this line is the main change to avoid double counting. everything else are just renames.